txmanager_integration_test.go

```diff
@@ -71,11 +72,22 @@ func TestTxManager_WithTransaction(t *testing.T) {
 	})
 
 	t.Run("AcrossRepoTxFailed_ThenRollback", func(t *testing.T) {
+		// start TxManager
 		txManager := txmanager.StartTxManager(db)
 		book2 := book{
 			Name: stringPointer("Biology"),
 		}
 
+		// ensure have author1 data
```

Comment:
@mariojuzar how about calling reset db method on every end of test? So every tests will have empty tables

ex.

```
t.Run("AcrossRepoTxFailed_ThenRollback", func(t *testing.T) {
    defer resetDb()
    ...
}

func resetDb() {
    models := []interface{}{
		book{},
		author{},
		authorBook{},
	}
	for _, v := range models {
		tableName := db.NewScope(v).TableName()
		err := db.Exec(fmt.Sprintf("TRUNCATE TABLE %s", tableName)).Error
		if err != nil {
			return err
		}
	}
}
```

Reply:
i see, that's good idea, will update it